src/widgetastic_patternfly/__init__.py

```diff
@@ -562,6 +562,9 @@ def is_disabled(self, text=None, **kwargs):
         """Check if an item is disabled"""
         if text:
             # Check if an item is disabled based on the text of that item
+            if isinstance(text, partial_match):
```

Comment:
Looks good!

Please add some assertions (or a new test case) to the unit test for this:

https://github.com/RedHatQE/widgetastic.patternfly/blob/master/testing/test_bootstrap_nav.py#L18